include/nuttx/lcd/lcd.h

```diff
@@ -137,6 +137,8 @@ struct lcd_planeinfo_s
    * buffers.
    */
 
+  int (*redraw)(FAR struct lcd_dev_s *dev);
```

Comment:
I think this should be moved and placed below `getarea`. A proper description should be added as well